### PR TITLE
Dropdown control evaluate text and logical operator nodes

### PR DESCRIPTION
**Pull request by FabriceIRANKUNDA**
## What Changed:

Added dropdown control on evaluate text node

## How to test:

Create evaluate text node and select an option

## Additional information:

Any other information that might be useful while reviewing.

### CODE REVIEW

**Comment by Knar33** [comment]
@FabriceIRANKUNDA #893 is a similar issue btw

File: packages/core/shared/src/nodes/text/EvaluateText.ts [hunk 1]
```diff
@@ -1,17 +1,17 @@
...
 import { stringSocket, triggerSocket } from '../../sockets'
 import { MagickNode, MagickWorkerInputs, WorkerData } from '../../types'
+import { DropdownControl } from '../../dataControls/DropdownControl'
 
 /** Pre-defined fewshot template */
 const fewshot = ''
 
 /** Information string for Evaluate Text component */
 const info =
-  'Takes a string input and evaluates it against the value specified in the text editor using the operation specified in the Operation Type property, then triggers the appropriate output based on the result. Valid values for the Operation Type: includes, not includes, equals, not equals, starts with, not starts with, ends with, not ends with.'
```
Comment (Knar33) [inline packages/core/shared/src/nodes/text/EvaluateText.ts:14]: Why did you revert the info?

**Review by Knar33 — COMMENTED** [review]
I tested every option and they all still work

**Review by lalalune — APPROVED** [review]
LGTM

**Pull request merged**